### ISSUE TITLE
[Bug]: (critical bug) random does not work after import vllm

### ISSUE THREAD

**Issue by yananchen1989** [vllm-project/vllm#8519]
### Your current environment

`vllm-0.6.1.post2`

### 🐛 Describe the bug

```
import random
print('===>====>', random.sample(list(range(100)), 5))  # randomization works

import torch
tokenizer = AutoTokenizer.from_pretrained(args.llm_name)
from vllm import LLM, SamplingParams
from vllm.lora.request import LoRARequest

print('===>====>', random.sample(list(range(100)), 5)) # randomization works


llm = LLM(model= args.llm_name, dtype='bfloat16', max_model_len=32000, 
        tensor_parallel_size=torch.cuda.device_count(), gpu_memory_utilization=0.95, 
            quantization="bitsandbytes", load_format="bitsandbytes", enforce_eager=True, 
            enable_lora= True if args.sft_path else False
        )    

print('===>====>', random.sample(list(range(100)), 5)) # randomization does not work anymore
```

this is my script. if you run it twice, you will find that after loading llm, the randomization at the last line,  will not work , yielding the same result each time you run.

what is the potential cause for this issue ?

### Before submitting a new issue...

- [X] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.

**Comment by simon-mo** [comment]
During the initialization process, we set the random seed. You can find it in worker's `init_device` method through the usage of `set_random_seed`.

**Comment by yananchen1989** [comment]
> During the initialization process, we set the random seed. You can find it in worker's `init_device` method through the usage of `set_random_seed`.

ok, thanks. anyway to eliminate the effect in the following randomzation process ?

**Comment by simon-mo** [comment]
`LLM(..., seed=None, ...)`
